config-controller/Makefile

```diff
@@ -0,0 +1,308 @@
+# VERSION defines the project version for the bundle.
```

Comment:
I think we should not use operator-sdk to generate the config-controller, instead using `kubebuilder`. operator-sdk basically wraps kubebuilder and adds mechanisms for application & package management, like registry, bundle which we do not need here.

We are just interested in generating a CRD, related manifests and the controller logic.
https://book.kubebuilder.io/

Reply:
My suggestion: re-generate the project with kubebuilder and migrate your code into the new base. It should be relatively fast, but by doing that we have a clean installation without operator-sdk.

Reply:
Thanks for this suggestion, I'll give this a go